src/services/homework/hooks/submissions.js

```diff
@@ -217,7 +217,7 @@ const noSubmissionBefore = (context) => {
 		|| ((submission.studentId || {})._id == context.params.account.userId));
 	if (submissionsForMe.length > 0) {
 		return Promise.reject(new Conflict({
-			message: `${submissionsForMe[0].studentId.firstName} ${submissionsForMe[0].studentId.lastName} hat bereits für dich abgegeben!`,
+			message: 'Benutzer hat bereits für sich abgegeben!',
```

Comment:
```suggestion
			message: 'Die Aufgabe wurde bereits abgegeben',
```
Please remove this Conflict and move the text into i18n

Reply:
OK, proposal accepted. I will take care about it.

Reply:
Is there any i18 support on the server?

Reply:
ah, good point, we should keep this as an error-response for future, for now - no.